index.html

```diff
@@ -259,13 +260,62 @@ <h2 id="display-constraints">
         </p>
         <p>
           The <code><a data-link-for="MediaTrackConstraintSet">cursor</a></code> constraint
-          allows an application to specify if the cursor should be included in the captured 
+          allows an application to observe if the cursor is be included in the captured
```

Comment:
Why this change?  Control over the inclusion of a cursor is a property we explicitly wanted to allow.  It doesn't affect source selection, so it's safe.  This change would revert an agreed change.

Reply:
Current state of the spec does not allow constraints, which is contradicting this statement about selecting cursor. It would make sense after #73 lands though.

Reply:
Agree we should revert this part.